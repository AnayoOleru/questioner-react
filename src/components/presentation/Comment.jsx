import React, { Fragment } from 'react';

const Comment = () => (
  <Fragment>
    <div className='Comment'>
      <div className='comment-area'>
        <div className='who-commented'>
          <h4>Daniel Adedeji</h4>
          <small>10:30am, 20th Dec 2018</small>
        </div>
        <div className='comment-details'>
            I thought the meetup was fake until i saw the reviews
        </div>
      </div>
    </div>
  </Fragment>
);

export default Comment;
