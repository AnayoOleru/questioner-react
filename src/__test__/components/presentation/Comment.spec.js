import React from 'react';
import { shallow } from 'enzyme';
import Comment from '../../../components/presentation/Comment';

describe('Comment Component', () => {
  let wrapper;
  beforeEach(() => {
    const props = {
      comment: 'herllo',
      createdon: 'thusday',
      firstname: 'Daniel',
      lastname: 'Geek'
    };
    wrapper = shallow(<Comment {...props} />);
  });
  test('Comment component should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
