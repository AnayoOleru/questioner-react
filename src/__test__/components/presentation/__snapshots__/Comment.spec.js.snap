// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment Component Comment component should render 1`] = `ShallowWrapper {}`;
